daemon/containerd/progress.go

```diff
@@ -337,3 +337,14 @@ func (combined combinedProgress) UpdateProgress(ctx context.Context, ongoing *jo
 	}
 	return nil
 }
+
+func showBlobProgress(desc ocispec.Descriptor) bool {
```

Comment:
Perhaps it's worth adding a GoDoc to this function.

It's clear _what_ it does, but not _why_ it does; a short description that it's used to decide whether or not progress should be shown, and why we're considering specific types should not show it.